docs/developer/pr-guidelines.md

```diff
@@ -0,0 +1,102 @@
+# PR Guidelines
+
+This document outlines the steps that need to be taken before merging any PR. In most cases, the only required action is creating a changelog entry (see below). However, when a PR affects the database schema, or the API, or service configuration, extra steps are required, and those are detailed below.
+
+The recommended way to use this document is to copy the relevant checklists below into the PR description, when appropriate, and make sure they are all checked before the PR is merged.
+
+## Changelog entries
+
+Every PR should add a new file in the appropriate subdirectory of `changelog.d`, containing just the text of the corresponding changelog entry. There is no need to explicitly write a PR number, because the `mk-changelog.sh` script (used on release) will add it automatically at the end. The name of the file does not matter, but it should be unique to avoid unnecessary conflicts (e.g. use the branch name).
+
+It is still possible to write the PR number manually if so desired, which is useful in case the entry is shared by multiple PRs. In that case, the script would leave the PR number reference intact, as long as it is at the very end of the entry, with no period afterwards and in brackets. It is also possible to use the pattern `##` to refer to the current PR number. This will be replaced throughout.
```

Comment:
I clarified how the PR number should be written, but I'm not quite sure what the other thing you're referring to is. If you can write a paragraph about that, I'll add it.

Reply:
https://github.com/wireapp/wire-server/pull/2476 for example was merged by "Merge", not "Squash", and the PR number was added in the PR writing the changelog entry.

Maybe after

> It is still possible to write the PR number manually if so desired, which is useful in case the entry is shared by multiple PRs

add a

> , or if the PR is merged with a merge commit, rather than a squash.

I'd move the part about `##` either before or in a new paragraph, because it doesn't make sense in the "non-squash case".

Reply:
Thanks, done.